tests/platform_tests/fwutil/fwutil_common.py

```diff
@@ -0,0 +1,228 @@
+import pytest
+import os
+import json
+import logging
+
+from copy import deepcopy
+
+from tests.common.utilities import wait_until
+
+logger = logging.getLogger(__name__)
+
+DEVICES_PATH="usr/share/sonic/device"
+TIMEOUT=3600
+REBOOT_TYPES = {
```

Comment:
@alexrallen suggest to follow the next code style:

```python
REBOOT_TYPES = {
    "cold": "reboot",
    "warm": "warm-reboot",
    "fast": "fast-reboot"
}
```

Reply:
Sure, done.

Reply:
Sure.